docs/source/llms/prompt-engineering/index.rst

```diff
@@ -293,6 +293,13 @@ can score the corresponding MLflow Model in a Python environment of your choosin
    For more information about deployment for real-time serving with MLflow,
    see the :ref:`instructions below <deploy-prompt-serving>`.
 
+Step 13: Perform metric-based evaluation of your model's outputs
+----------------------------------------------------------------
+If you'd like to assess your model's performance on specific metrics, MLflow provides the :py:func:`mlflow.evaluate()`
```

Comment:
Maybe we should inline some code showing how to evaluate the model that loaded from Step 12 with `mlflow.evaluate()`, cross referencing a big doc isn't very user friendly.

Reply:
I think a very quick example would work well here too :)

Reply:
Will add one in soon! I just discovered a couple of bugs in the prompt engineering UI code that makes it difficult to construct an example at the moment, but working to get those fixed haha